modules/claims_api/config/schemas/v2/526.json

```diff
@@ -136,10 +139,9 @@
           }
         },
         "currentVaEmployee": {
-          "type": ["boolean", "null"],
+          "type": ["boolean"],
```

Comment:
I think this should change to just `"type": "boolean"` , right? No array there anymore.

Reply:
Good catch, thanks!